### PR TITLE
Added Costa Rica #1863

### PR DESCRIPTION
**Pull request by Vishrut19**
Added Costa Rica #1863

### CODE REVIEW

**Comment by Vishrut19** [comment]
@BaseMax @VictorAraica @smriad Kindly review this PR

**Review by jbampton — COMMENTED** [review]
Files and folders should use `-`

File: doc/source/costa_rica.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+Although it is true that there are more than 100 volcanoes in this country, only five of them are active and although there are other inactive volcanoes that are also worth knowing, the Poas Volcano may be one of the most interesting, since it can be seen its immense crater and is one of the largest in the world, it is also located in a national park with a lush forest.
+
+## Peace Waterfall
+
+Very close to the Poas is the most impressive waterfall in the country, but it is located inside a private enclosure where they have already built a zoo, so the entrance fee is 48 dollars. It is not recommended for the price, especially in a country where you can see a diverse fauna that is free.
+
+## San Jose, the capital of Costa Rica
+
+In San José you can learn a little about the history of the country, mingle with the local people and immerse yourself in their culture. It is advisable to take a walk through the center, visit the National Theater, see the majestic Metropolitan Cathedral, the Central Market, a colorful Church called Nuestra Señora de la Merced, stroll through one of its many museums and also in a good place to have a drink. , eat or go dancing.
```
Comment (jbampton) [inline doc/source/costa_rica.md:55]:
```suggestion
In San José you can learn a little about the history of the country, mingle with the local people and immerse yourself in their culture. It is advisable to take a walk through the center, visit the National Theater, see the majestic Metropolitan Cathedral, the Central Market, a colorful Church called Nuestra Señora de la Merced, stroll through one of its many museums and also in a good place to have a drink, eat or go dancing.
```

**Review by jbampton — CHANGES_REQUESTED** [review]
Costa Rica is a country

**Comment by jbampton** [comment]
Broken links

**Review by BaseMax — COMMENTED** [review]
Thank you for your PR.

**Comment by Vishrut19** [comment]
Changes have been made @jbampton

File: doc/source/costa-rica.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+Although it is true that there are more than 100 volcanoes in this country, only five of them are active and although there are other inactive volcanoes that are also worth knowing, the Poas Volcano may be one of the most interesting, since it can be seen its immense crater and is one of the largest in the world, it is also located in a national park with a lush forest.
+
+## Peace Waterfall
+
+Very close to the Poas is the most impressive waterfall in the country, but it is located inside a private enclosure where they have already built a zoo, so the entrance fee is 48 dollars. It is not recommended for the price, especially in a country where you can see a diverse fauna that is free.
+
+## San Jose, the capital of Costa Rica
+
+In San José you can learn a little about the history of the country, mingle with the local people and immerse yourself in their culture. It is advisable to take a walk through the center, visit the National Theater, see the majestic Metropolitan Cathedral, the Central Market, a colorful Church called Nuestra Señora de la Merced, stroll through one of its many museums and also in a good place to have a drink. , eat or go dancing.
```
Comment (jbampton) [inline doc/source/costa-rica.md:55]:
```suggestion
In San José you can learn a little about the history of the country, mingle with the local people and immerse yourself in their culture. It is advisable to take a walk through the center, visit the National Theater, see the majestic Metropolitan Cathedral, the Central Market, a colorful Church called Nuestra Señora de la Merced, stroll through one of its many museums and also in a good place to have a drink, eat or go dancing.
```

**Review by jbampton — APPROVED** [review]

**Pull request merged**